### ISSUE TITLE
Contents of comments in Word 365 with NVDA+alt+c or elements list.

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14985]
### Steps to reproduce:

1. Open Word and type some text
2. Press control+alt+m to add a comment
3. Type something, then press alt+enter to add it
4. Press escape to return to the text.
5. Locate the comment, and press NVDA+alt+c to read the comment.
6. Press NVDA+spacebar to put NVDA in browse mode
7. Press NVDA+f7 to open the elements list
8. Press alt+a to jump to annotations and read the annotation for the comment.

### Actual behavior:

Using both NVDA+alt+c and the elements list, NVDA reads the user who added the comment and the time the comment was added:

Comment: Comment thread started by Quentin Christensen by Quentin Christensen on 08 June 2023, 9:58 AM

But the actual text of the comment is not read.

### Expected behavior:

NVDA should first read the text of the comment, and then read any associated info such as who left the comment and when.

#### NVDA version:
NVDA 2023.1 installed
#### Windows version:

Windows 11 (64-bit) Version: 22H2, Build: 22621.1778 

#### Name and version of other software in use when reproducing the issue:

Office 365 (64-bit) Version: 16.0.16327.20248

**Comment by michaelDCurran** [comment]
Here we are talking about comment threads. I.e. a comment replies. Is it okay to just announce the root comment?

**Comment by Qchristensen** [comment]
Yes I think just announcing the root comment would be fine.  I'm not sure how deep people go with comments - I know when I've used them it's mostly only one level with occasionally a reply.  It does announce how many replies there are so that already works, it's just that it is reading out every other bit of information except the comment itself.

**Comment by Qchristensen** [comment]
For anyone encountering this, the workaround I've found is:

1) Press A or navigate to where the comment is
2) Press alt+r, n to read the next comment
3) Press escape to return to the text.

**Comment by Sylvie42** [comment]
When you add a response to a comment, NVDA does not inform you that there is a response.

You can only be informed about that response when you display the list of annotations with insert+F7 and alt+A or when you display the list of comments in the MS ribbon.

NVDA tells you then that there is a response but it only tells you the author's name and the date, not the content of the comment.

At last, in browse mode, typing a to reach next comment does not bring you to the comment response.

Last note: I am not sure if it is linked to the way comments are rendered by NVDA, bu when you write a comment, or when you respond to a comment, you cannot read what you type in braille. If you want to read your text in braille, you have to press tab and shift+tab. In that situation, the text is refreshed and you can read it. Just tell me if it belongs to this issue or if I should create another one.

**Comment by Qchristensen** [comment]
This is still present in:

Windows 11 (64-bit) Version: 23H2, Build: 22631.3155 
Office 365 (64-bit) Version: 16.0.17328.20068 
NVDA 2024.1 Beta 7

And in this setup, the workaround above (alt+r, n) works, and another workaround I've found is if Use UIA in Word is set to "Only when necessary", pressing NVDA+alt+c on a comment reads ONLY the comment.

In every real-world situation where I have encountered comments, I have only ever had to work with one layer of comments (ie not conversations) and only from one user, so this approach would be much more efficient in that case.

As and when we do improve the reading experience, I wonder if we could consider adding a setting somewhere to control how much information is reported for comments?  Knowing who made which comments is useful where a whole team is sending around one document with multiple suggestions, but in every case I've used it, it's only ever been one person making comments, so NOT having it read their name and the time they made the comment would be much more preferable (to me) for the most part.

**Comment by manish10** [comment]
I work with documents with long comment conversations all the time. it will be most useful to read the latest response on a comment rather than the root comment with the hot key. for now, I use the alt+r+n but being able to just read the latest response on any comment will be super useful. new feature requests would be to go to only comments with responses (can file separately if is not already a feature I am aware of)
